docs/docs/nlu-based-assistants/rules.mdx

```diff
@@ -192,5 +192,5 @@ how the form is defined, ignoring rules. Rules become applicable again if:
 
 - the form fills all required slots
 - the form rejects its execution (see
-  [Handling Unhappy Paths](./forms.mdx#writing-stories--rules-for-unhappy-form-paths) for
+  [Conversation Repair](./forms.mdx#writing-stories--rules-for-conversation-repair-cases) for
```

Comment:
wouldn't have thought to go back and do the change the NLU-based assistants, but I agree it makes sense!